app/utils/Messages.scala

```diff
@@ -204,4 +206,30 @@ object Messages {
 
     xml
   }
+
+  def generateIE028Message(): NodeSeq = {
+    val xml =
+      <CC028A>
+        <SynIdeMES1>{Strings.alpha(4)}</SynIdeMES1>
+        <SynVerNumMES2>{Strings.numeric(1)}</SynVerNumMES2>
+        <MesSenMES3>{Strings.alphanumeric(1, 35)}</MesSenMES3>
```

Comment:
Can you create a sub task on CTDA-390 to remove all uses of MesSenMES3 (where relevant) in the test support API?

Reply:
Done 👍